packages/@storefront/core/test/unit/services/urlUtils.ts

```diff
@@ -16,6 +16,29 @@ suite('URL Service', ({ expect, spy, stub }) => {
   });
 
   describe('static', () => {
+    describe('getBaseUri', () => {
+      it('should return the base URI', () => {
+        win.document = { baseURI: 'foo' };
+
+        expect(Utils.getBaseUri()).to.eq('foo');
+      });
+
+      it('should extract the `href` property from the <base> element', () => {
+        win.document = {
+          querySelector: stub().withArgs('base').returns({ href: 'bar' }),
+        };
+
+        expect(Utils.getBaseUri()).to.eq('bar');
+      });
+
+      it('should fall back to the current location', () => {
+        win.location = { href: 'baz' };
+        win.document = { querySelector: spy() };
```

Comment:
This `spy` is not being used. You should probably have an assertion using it.

Reply:
Resolved.